### PR TITLE
generate_sbom: add help option

### PR DESCRIPTION
**Pull request by adrianschroeter**
Also add the option --tar for container tar balls. But without enforcing it to stay backward compatible.

### CODE REVIEW

**Comment by adrianschroeter** [comment]
@msmeissn please review the help text

File: generate_sbom [hunk 1]
```diff
@@ -562,8 +563,13 @@ while (@ARGV && $ARGV[0] =~ /^-/) {
     $isdir = 1;
   } elsif ($opt eq '--rpmmd') {
     $rpmmd = 1;
+  } elsif ($opt eq '--tar') {
+    $istar = 1;
   } elsif ($opt eq '--format') {
     $format = shift @ARGV;
+  } elsif ($opt eq '--help') {
+    echo_help();
+    exit(1);
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply generate_sbom:572]: From [the documentation](https://perldoc.perl.org/functions/exit): _The only universally recognized values for EXPR are 0 for success and 1 for error;_

File: generate_sbom [hunk 2]
```diff
@@ -572,7 +578,45 @@ while (@ARGV && $ARGV[0] =~ /^-/) {
...
+  --intoto
+     Can be used optional to wrap the generated data into in-toto.io
+     specified format.
+
+Supported content
+=================
+
+  --dir DIRECTORY
+     A Linux installation to scan. This can be any directory structure
+     which provides an installed system.
```
Comment (msmeissn) [inline generate_sbom:606]: The RPM database of the system below DIRECTORY will be evaluated, also all files will be referenced in the SBOM.

Comment (adrianschroeter) [reply generate_sbom]: changed

**Pull request merged**